job-server/src/spark.jobserver/io/JobSqlDAO.scala

```diff
@@ -5,11 +5,20 @@ import java.io.{FileOutputStream, BufferedOutputStream, File}
 import java.sql.Timestamp
 import org.joda.time.DateTime
 import org.slf4j.LoggerFactory
+import scala.slick.driver.JdbcProfile
 import scala.slick.jdbc.meta.MTable
-
+import scala.reflect.runtime.universe
 
 class JobSqlDAO(config: Config) extends JobDAO {
-  import scala.slick.driver.H2Driver.simple._
+  val slickDriverClass = getOrElse(config.getString("spark.jobserver.sqldao.slick-driver"),
```

Comment:
I know we have getOrElse, but I think the pattern of putting defaults in `reference.conf` is more standard now and we should follow it.